### PR TITLE
add ebooks and job sites

### PR DESCRIPTION
**Pull request by rkimera94**

### CODE REVIEW

File: site/lists/ebooksData.scroll [hunk 1]
```diff
@@ -2,3 +2,8 @@ importOnly
 commaTable
  title,titleLink,author,year,publisher
  Introduction to the Theory of Programming Languages,https://bertrandmeyer.com/2022/09/28/introduction-theory-programming-languages-full-book-now-freely-available/,Bertrand Meyer,2022,Prentice Hall International Series in Computer Science
+ Programming Fundamentals,https://openlibrary-repo.ecampusontario.ca/jspui/bitstream/123456789/692/3/Programming-Fundamentals-1570222270.pdf,KENNETH LEROY BUSBEE,2008
```
Comment (breck7) [inline site/lists/ebooksData.scroll:5]: author name should not be in all caps

File: site/lists/ebooksData.scroll [hunk 2]
```diff
@@ -2,3 +2,8 @@ importOnly
...
+ Python Basics: A Practical Introduction to Python 3,https://static.realpython.com/python-basics-sample-chapters.pdf,David Amos,2012–2020
+ Learning Python,https://cfm.ehu.es/ricardo/docs/python/Learning_Python.pdf,Mark Lutz,2009
+ How to Code in Go,https://assets.digitalocean.com/books/how-to-code-in-go.pdf,Mark Bates,2020
+ The Go Programming Language,http://www.cs.uniroma2.it/upload/2017/TSC/The%20Go%20Programming%20Language.pdf,Alan A. A. Donovan,2009
```
Comment (breck7) [inline site/lists/ebooksData.scroll:9]: Is it really A. A. or is that a typo?

File: site/lists/jobs.scroll [hunk 3]
```diff
@@ -27,6 +27,8 @@ pipeTable
  Hubstaff|https://talent.hubstaff.com/
  Berlinstartupjobs|https://berlinstartupjobs.com/
  Vanhack|https://vanhack.com/jobs
+  Braintrust|https://www.usebraintrust.com/
```
Comment (breck7) [inline site/lists/jobs.scroll:30]: extra space

File: site/lists/ebooksData.scroll (hunk 1 above)
Comment (rkimera94) [reply site/lists/ebooksData.scroll:5]: okay, let me fix it

File: site/lists/jobs.scroll (hunk 3 above)
Comment (rkimera94) [reply site/lists/jobs.scroll:30]: Screenshot 2022-10-04 at 17 23 36

File: site/lists/ebooksData.scroll (hunk 2 above)
Comment (rkimera94) [reply site/lists/ebooksData.scroll:9]: https://github.com/breck7/pldb/pull/174#discussion_r986938400
Its A.A  according  to the book

File: site/lists/jobs.scroll (hunk 3 above)
Comment (rkimera94) [reply site/lists/jobs.scroll]: @breck7  I have made the changes

Comment (breck7) [reply site/lists/jobs.scroll]: great work!

**Pull request merged**